### PR TITLE
Apply backend validation for User First and last name

### PR DESCRIPTION
**Pull request by karpagamNallasamy**
Note : API response alert have confirmed with Artur

### CODE REVIEW

File: api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs [hunk 1]
```diff
@@ -71,5 +71,35 @@ public static bool IsPasswordValidForRequiredCharactors(string password)
       Match match = regex.Match(password);
       return match.Success;
     }
+
+    public static bool IsUserNameValid(string Name)
+    {
+      if (string.IsNullOrWhiteSpace(Name))
```
Comment (VijayHirudayasamy-bc) [inline api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs:77]: Can we add required field attributes to the model to avoid this check

**Review by VijayHirudayasamy-bc — CHANGES_REQUESTED** [review]

**Comment by karpagamNallasamy** [comment]
Resolved as per comments. Committed and pushed into this same PR.

Please take on look.

________________________________

File: api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs [hunk 2]
```diff
@@ -24,12 +24,12 @@ public static bool IsEmailFormatValid(string emailaddress)
 
     public static bool IsEmailLengthValid(string email)
     {
-     try
+      try
```
Comment (VijayHirudayasamy-bc) [inline api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs:27]: please fix the formatting

File: api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs [hunk 3]
```diff
@@ -71,5 +71,24 @@ public static bool IsPasswordValidForRequiredCharactors(string password)
...
+    public static bool IsUserNameValid(string Name)
+    {
+      Regex regex = new Regex(RegexExpression.VALID_USER_NAME);
+      Match match = regex.Match(Name);
+      return match.Success;
+    }
+
+    public static bool IsUserNameLengthValid(string Name)
+    {
+      if (Name.Length <= 1)
```
Comment (VijayHirudayasamy-bc) [inline api/CcsSso.Shared/Domain/Helpers/UtilityHelper.cs:84]: Can we simplify this methods as follows 
return Name.Length <= 1 ? false : true;
Please check and let me know your thought. Please test it

**Review by VijayHirudayasamy-bc — CHANGES_REQUESTED** [review]

**Review by VijayHirudayasamy-bc — APPROVED** [review]
Approved. Please follow up to deliver the PR.
Hope the QA Buddy check has been done

**Review by VijayHirudayasamy-bc — APPROVED** [review]

**Pull request merged**